.github/workflows/pr-verify.yaml

```diff
@@ -1,19 +1,19 @@
-name: PR verify
+name: PR title verifier
 
 on:
   pull_request_target:
     types: [opened, edited, synchronize, reopened]
 
-permissions:
-  checks: write # Allow access to checks to write check runs.
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need this?

We have many other actions using checkout, but none of them needs this

Reply:
Oh, sorry. This was used to write to PR, but forgot to erase it. Thanks

Reply:
No problem